catalog/spatial/spatial-app/src/main/resources/features.xml

```diff
@@ -133,9 +133,9 @@
     <feature name="webservice-gazetteer" version="${project.version}"
              description="Gazetteer service utilizing the Geonames.org web service.">
         <feature>spatial-geocoding-endpoint</feature>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
-        <bundle>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-core/${jackson.version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson.version}</bundle>
+        <bundle>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson.version}</bundle>
```

Comment:
✏️ The jackson feature is defined twice. One of them can be removed.
https://github.com/codice/ddf/blob/0af70deb885c1b8734fcc90fab21a8421996ebf3/catalog/catalog-app/src/main/resources/features.xml#L95-L99
https://github.com/codice/ddf/blob/0af70deb885c1b8734fcc90fab21a8421996ebf3/features/security/src/main/feature/feature.xml#L405-L409

Reply:
Done